packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -0,0 +1,33 @@
+import { BufferComposer, ComposableBuffer } from '../../buffer/buffer_composer'
+import BigNumber from 'bignumber.js'
+
+// Disabling no-return-assign makes the code cleaner with the setter and getter */
+/* eslint-disable no-return-assign */
+
+/**
+ * CreateLoanScheme DeFi Transaction
+ */
+export interface CreateLoanScheme {
+  ratio: number
```

Comment:
add actual length information like this? https://github.com/DeFiCh/jellyfish/blob/f4ea2b399fe2ee799dedd4bb83ea8639e8178721/packages/jellyfish-transaction/src/script/dftx/dftx_icxorderbook.ts#L11

Reply:
Ya, should add.  I am going to add to other the dftx PRs too.